reference/toaztr.v1.yaml

```diff
@@ -1114,32 +970,191 @@ paths:
+        - OAuth:
+            - cases/read
+      parameters:
+        - $ref: '#/components/parameters/page[size]'
+        - $ref: '#/components/parameters/page[after]'
+        - $ref: '#/components/parameters/page[before]'
+      tags:
+        - Cases
+    post:
+      summary: Creates simulations
+      operationId: postSimulation
+      responses:
+        '200':
+          description: OK
           content:
             application/json:
               schema:
-                $ref: ../models/ErrorResponse.v1.json
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/SimulationResource'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
         '500':
-          description: Internal server error
+          $ref: '#/components/responses/500'
+      description: Creates a simulation for this case and plan-parameters
+      parameters: []
+      security:
+        - OAuth:
+            - cases/write
+      tags:
+        - Cases
+  '/v1/cases/{caseId}/plans-parameters/{planId}/simulations/{id}':
+    parameters:
+      - $ref: '#/components/parameters/caseId'
+      - $ref: '#/components/parameters/planId'
+      - $ref: '#/components/parameters/id'
+    get:
+      summary: Returns simulations by ID
+      tags:
+        - Cases
+      responses:
+        '200':
+          description: OK
+          headers:
+            ETag:
+              schema:
+                type: string
+              description: Entity tag of the resource
           content:
             application/json:
               schema:
-                $ref: ../models/ErrorResponse.v1.json
-      summary: Creates plan-parameters
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/SimulationResource'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+      operationId: getSimulationById
+      description: Fetch a simulation by ID
+      security:
+        - OAuth:
+            - cases/read
+    delete:
+      summary: Deletes simulations based on ID
+      operationId: deleteSimulation
+      responses:
+        '200':
+          description: OK
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '409':
+          $ref: '#/components/responses/409'
+        '500':
+          $ref: '#/components/responses/500'
+      description: Deletes a simulation by ID
+      security:
+        - OAuth:
+            - cases/write
+      tags:
+        - Cases
 servers:
   - description: Production server
```

Comment:
Here, do we want to add dev and sandbox ?

Reply:
dev I don't think so but we could add sandbox
However I don't know how it works given we also duplicated the PRD urls in the OAuth information.
I agree it would be interesting, but maybe we should think more about how it should be integrated before